docs: reduced wcag disclaimer (#2936)
nl-design-system/documentatie: docs/wcag/_wcag-disclaimer.md

> ## Wettelijke verplichting

Reviewer: Ik denk dat geen in de titel hier belangrijk is. 

```suggestion
## Geen wettelijke verplichting
```

Reviewer 2: Misschien is het wel nog vriendelijker om te gaan voor de paginatitel van de DigiToegankelijk-pagina:

```suggestion
## Wat is verplicht?
```

Author: Voorstel van @Robbert overgenomen.

> De richtlijnen zijn geen wettelijke verplichting of vervanging voor WCAG. Ze zijn een praktische uitleg met voorbeelden die helpen bij het toegankelijk inzetten van NL Design System. Meer informatie of de wettelijke verplichting staat op de pagina [wat is verplicht van DigiToegankelijk](https://www.digitoegankelijk.nl/wetgeving/wat-is-verplicht).

Reviewer 2: ```suggestion
Ons doel is om praktische uitleg en voorbeelden te geven die helpen bij het maken van toegankelijke websites en webapplicaties. We doen dat op basis van een interpretatie van de WCAG 2.2 specificatie, en we beperken ons tot veelvoorkomende situaties.

De richtlijnen zijn dus geen wettelijke verplichting of vervanging voor WCAG. Meer informatie of de wettelijke verplichting staat op de pagina ["Wat is verplicht?" van DigiToegankelijk](https://www.digitoegankelijk.nl/wetgeving/wat-is-verplicht).
```
